ZTNA APIs [Public Preview] (#21578)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/networkaccess-forwardingoptions.md

---
title: "forwardingOptions resource type"

> description: "Defines the traffic forwarding options for the tenant. The forwardingOptions object contains the skipDnsLookupState settings, which determine whether DNS lookup will be performed at the client, allowing M365 traffic to be forwarded directly to Azure Front Door, or if DNS resolution will be conducted at the Edge. This operation provides control over the traffic flow and DNS resolution strategy within the network infrastructure."

Reviewer: 160 char limit
```suggestion
description: "Defines the traffic forwarding options for the tenant. "
```

author: "Moti-ba"
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: resourcePageType
---

# forwardingOptions resource type

Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Defines the traffic forwarding options for the tenant. The forwardingOptions object contains the skipDnsLookupState settings, which determine whether DNS lookup will be performed at the client, allowing M365 traffic to be forwarded directly to Azure Front Door, or if DNS resolution will be conducted at the Edge. This operation provides control over the traffic flow and DNS resolution strategy within the network infrastructure

Reviewer: ```suggestion
Defines the traffic forwarding options for the tenant. The forwardingOptions object contains the **skipDnsLookupState** settings, which determine whether DNS lookup will be performed at the client, allowing Microsoft 365 traffic to be forwarded directly to the Front Door, or if DNS resolution will be conducted at the edge. This operation provides control over the traffic flow and DNS resolution strategy within the network infrastructure.
```

Inherits from [microsoft.graph.entity](../resources/entity.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Get forwardingOptions](../api/networkaccess-forwardingoptions-get.md)|[microsoft.graph.networkaccess.forwardingOptions](../resources/networkaccess-forwardingoptions.md)|Read the properties and relationships of a [microsoft.graph.networkaccess.forwardingOptions](../resources/networkaccess-forwardingoptions.md) object.|
|[Update forwardingOptions](../api/networkaccess-forwardingoptions-update.md)|[microsoft.graph.networkaccess.forwardingOptions](../resources/networkaccess-forwardingoptions.md)|Update the properties of a [microsoft.graph.networkaccess.forwardingOptions](../resources/networkaccess-forwardingoptions.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|Id, Inherited from [microsoft.graph.entity](../resources/entity.md).|
> |skipDnsLookupState|microsoft.graph.networkaccess.status|The forwardingOptions control manages the various traffic forwarding options.The possible values are: `enabled`, `disabled`.|

Reviewer: ```suggestion
|id|String|Identifier. Inherited from [microsoft.graph.entity](../resources/entity.md).|
|skipDnsLookupState|microsoft.graph.networkaccess.status|The forwardingOptions control manages the various traffic forwarding options. The possible values are: `enabled`, `disabled`.|
```
